crowdfunding_compassion/models/crowdfunding_project.py

```diff
@@ -77,7 +80,13 @@ def create(self, vals):
             }
         )
         res.event_id = event
+
         res.add_owner2participants()
+
+        self.env['utm.campaign'].create({
+            "name": vals.get('name'),
+        })
```

Comment:
This should be removed. The `_inherits` take care of it.
```suggestion
```

Reply:
yes I added the inherits but forgot to remove the create..